src/api/app/views/webui/comment/_content.html.haml

```diff
@@ -8,6 +8,15 @@
       wrote
       = link_to("#comment-#{comment.id}", title: I18n.l(comment.created_at.utc), name: "comment-#{comment.id}") do
         = render TimeComponent.new(time: comment.created_at)
+
+      - if User.session && User.session.submitted_reports.where(reportable: comment).empty?
```

Comment:
We can encapsulate this line in a helper because we're using it for packages also

Reply:
Done. This logic is now part of ReportPolicy